### ISSUE TITLE
CheckBox- Accessibility violation - ARIA tooltip nodes must have an accessible name

### ISSUE THREAD

**Issue by divyajindel** [Sage/carbon#4628]
### Current behaviour

The Checkbox has an accessibility violation "Ensures every ARIA tooltip node has an accessible name" 
for example - https://carbon.sage.com/?path=/story/checkbox-validations--group-checkbox-string-validation

### Expected behaviour

There are no accessibility violations.

### CodeSandbox or Storybook URL

https://carbon.sage.com/?path=/story/checkbox-validations--group-checkbox-string-validation

### Carbon Version

latest

### What browsers are you seeing the problem on?

Firefox, Chrome, Safari, Microsoft Edge

### What Operating System are you seeing the problem on?

MacOS

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.

**Comment by samtjo** [comment]
FE-4632

**Comment by ZhuoyuJin** [comment]
No corresponding accessibility issue found in Checkbox component, ticket closed.
